modules/checklist/steps/step-base.php

```diff
@@ -208,4 +208,10 @@ public function get_step_initial_progress() : array {
 	private function set_step_progress() : void {
 		$this->module->set_step_progress( $this->get_id(), $this->user_progress );
 	}
+
+	public function page_exists( $params ) : bool {
```

Comment:
Does this function need to be in the base?

Reply:
Won't we use it in the 404 page?

Reply:
we can move it when needed